src/containers/HomeTemplate/_components/infoElearning/index.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faGraduationCap,
+  faBookmark,
+  faBook,
+} from "@fortawesome/free-solid-svg-icons";
+import CountUp from "react-countup";
+import VisibilitySensor from "react-visibility-sensor";
+
+export default function InfoElearning() {
+  return (
+    <div
+      className="my-5 info-elearning"
+      style={{ backgroundImage: 'url("img/bg-info-elearning.jpg")' }}
+    >
+      <div
+        className="ie-overlay"
+        style={{ backgroundImage: 'url("img/bg-2.png")' }}
+      ></div>
+      <div className="ie-content row">
+        <div className="icon-group col-sm-12 col-md-4">
+          <FontAwesomeIcon icon={faBookmark} className="icon" />
+          <div className="name-icon">Teachers</div>
+          <CountUp start={0} end={20} duration={5}>
+            {({ countUpRef, start }) => (

```

Comment:
create a function and use it in here, please don't use anonymous function like this